tidy3d/components/data/unstructured/triangular.py

```diff
@@ -589,6 +591,7 @@ def plot(
         shading: Literal["gourand", "flat"] = "gouraud",
         cbar_kwargs: Optional[dict] = None,
         pcolor_kwargs: Optional[dict] = None,
+        transpose: bool = False,
```

Comment:
I *definitely* tested the changes to this function, but I'm having trouble remembering exactly which notebook example invokes this function.  (It might have been the [MachZehnderModulator.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/MachZehnderModulator.ipynb) example.)